packages/knip/test/plugins/react-router.test.ts

```diff
@@ -25,3 +24,20 @@ test('Find dependencies with the react-router plugin', async () => {
     files: isWindows ? 1 : 0,
   });
 });
+
+test('Find dependencies with the react-router plugin [with entries]', async () => {
+  const cwd = resolve('fixtures/plugins/react-router-with-entries');
+  const { counters } = await main({
+    ...baseArguments,
+    cwd,
+  });
+
+  assert.deepEqual(counters, {
+    ...baseCounters,
+    processed: 11,
+    total: 11,
+    // There is a bug with routes that include () on Windows so they will not be found there, revert when
+    // the bug is fixed
+    files: isWindows ? 1 : 0,
```

Comment:
Let's keep the new fixture folder clean so we can focus on the issue at hand. Then we don't need to specify any of this Windows-related trouble here.